From e74416675cf9524720e22f13d2a7a89e23ba7d41 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Fri, 19 Oct 2018 12:21:59 +0200
Subject: [PATCH 8/8] Merge android-9.0.0_r12

commit 47e38865645ffe8bf0a535ace66c541a8af56b5d
Merge: d71f0f1d9 d93f99318
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Mon Jul 23 23:06:46 2018 +0000

    Merge cherrypicks of [4607809, 4607810, 4609855, 4608092, 4608093, 4607931, 4607932, 4609887, 4609888, 4609907, 4609873, 4609874, 4609927, 4609928, 4609967, 4609968, 4609969, 4609970, 4609971, 4607550, 4607551, 4607552, 4607553, 4607554, 4607555, 4607556, 4607557, 4607558, 4607559, 4609929, 4608094, 4608095, 4608096, 4608097, 4608098, 4608099, 4610018, 4610019, 4610047] into pi-dr1-release

    Change-Id: I1188a346e70fa3402a1ce5bf61321fbd2781a78f

commit d93f993189f1d943bfb395dd3a043b8287350e79
Author: Thierry Strudel <tstrudel@google.com>
Date:   Thu Jul 19 18:49:15 2018 -0700

    metricslogger: fix ACTION_BATTERY_CAUSED_SHUTDOWN not matching proto

    Bug: 111664962
    Test: "pixelstats_client -s" uses the right ATOM value
    Change-Id: I2944c6d9f79298ce88812fad218552b92afee97a
    Signed-off-by: Thierry Strudel <tstrudel@google.com>
    (cherry picked from commit ae2e19dd96e47a084eac09c3b3095b3bef427415)

commit d71f0f1d9d72b3373d0c7977e99f307f1396d2c4
Merge: 89579377d 93d837f3a
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Thu Jun 21 03:05:22 2018 +0000

    Snap for 4853718 from 93d837f3a90acec007647f21ed4573f044fa6f1e to pi-dr1-release

    Change-Id: I1c7c91b4258e5dc4c374b3a205d4d733586432be

commit 93d837f3a90acec007647f21ed4573f044fa6f1e
Author: Doheon Lee <doheon1.lee@lge.com>
Date:   Tue Jun 19 15:23:21 2018 +0900

    Determine product partition path on build time

    Path of product partitoin can be set as /product or /system/product
    whether generate extra product partition or not.
    Substitute %PRODUCT% to relevant path to know linker which path should
    search and permit.

    Bug: 110286945
    Test: m -j # Check /system/etc/ld.config.$(PLATFORM_VNDK_VERSION).txt
    Change-Id: I6ca177d0c9c5af00ad821879fece40848331fc8d
    Merged-In: I6ca177d0c9c5af00ad821879fece40848331fc8d
    (cherry picked from commit cccad0bf8461ab575a5861f9bc68c9f8f69db582)

commit c4cf62f4b4e9186a897e943b2419fb9a13739c1b
Author: Inseob Kim <inseob@google.com>
Date:   Mon Jun 18 13:21:45 2018 +0900

    Add /system/product/* paths to permitted paths

    As linker doesn't resolve paths in permitted paths, /system/product
    variants should be added to support devices having product partition
    under /system.

    Bug: 110286945
    Test: m -j succeeds on taimen and libraries under /system can dlopen
    libraries under /system/product/apps

    Change-Id: Icd102d44511702e4ec66c07a367b59c3d9700a44
    Merged-In: Icd102d44511702e4ec66c07a367b59c3d9700a44
    (cherry picked from commit 3918936b9e7fee51828d285a1217c7202d5cb2d1)

commit 89579377df669b2f901885ba5aad5f0710a8d8bd
Merge: 62d469bcc ee81388ed
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Tue Jun 19 03:00:50 2018 +0000

    Snap for 4848184 from ee81388edb86973c81a26b0b97e957fa05425c64 to pi-dr1-release

    Change-Id: If488ee53da3cd2c85df0fa5514d8034941ceadaf

commit ee81388edb86973c81a26b0b97e957fa05425c64
Merge: 95d314be3 9a95028ea
Author: TreeHugger Robot <treehugger-gerrit@google.com>
Date:   Mon Jun 18 23:35:25 2018 +0000

    Merge "Revert "healthd: add stem to health@2.0-service.override [DO NOT MERGE]"" into pi-dev

commit 9a95028eac3bb01a4c04012a62a4588d9eed5558
Author: Yifan Hong <elsk@google.com>
Date:   Mon Jun 18 21:22:49 2018 +0000

    Revert "healthd: add stem to health@2.0-service.override [DO NOT MERGE]"

    This should have been done with actual fixes to sepolicy and rc scripts.

    This reverts commit 299276785787a35effa829f3f9d507cbde266dea.

    Reason for revert: broke the build

    Bug: 110228707

    Change-Id: I23944d044dd2f87e6c1033ed7a4c7cf34f16247e

commit 95d314be388b1cb28f4987937078a190d3da5c76
Merge: 74e8061fd 299276785
Author: Yifan Hong <elsk@google.com>
Date:   Mon Jun 18 20:15:24 2018 +0000

    Merge "healthd: add stem to health@2.0-service.override [DO NOT MERGE]" into pi-dev

commit 62d469bccc5de85756b28c8378ec0792b829220a
Merge: a612ed76f 74e8061fd
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Sun Jun 17 03:05:36 2018 +0000

    Snap for 4845430 from 74e8061fd04c962d6e002109de57795eabd8f04b to pi-dr1-release

    Change-Id: I2e5a4ce723a72525ed6c7426a9a9d2e166fd4a46

commit 299276785787a35effa829f3f9d507cbde266dea
Author: Yifan Hong <elsk@google.com>
Date:   Thu Jun 14 15:12:26 2018 -0700

    healthd: add stem to health@2.0-service.override [DO NOT MERGE]

    The override module should have the same name as the -service
    module; the only difference is that healthd is installed or not.
    Without this change, the binary is completely useless due to
    incorrect rc files and sepolicy file_contexts.

    This allows emulator to use this module correctly.

    Bug: 110228707

    Test: builds on emulator, boots
    Change-Id: Id2062925ec3567c4fde70358184bb790541bf83b

commit 74e8061fd04c962d6e002109de57795eabd8f04b
Merge: f514f6f66 b8371d532
Author: TreeHugger Robot <treehugger-gerrit@google.com>
Date:   Thu Jun 14 17:29:54 2018 +0000

    Merge "Populate recovery DTBO offset correctly" into pi-dev

commit f514f6f66aba10f2668481a4d643c7983a0ea93a
Author: Hridya Valsaraju <hridya@google.com>
Date:   Wed May 30 10:57:18 2018 -0700

    Correct comment in boot image header

    Test: make
    Bug: 80207223
    Change-Id: I88cf7683e036ff43bbe3ff5418519c7e35e08b79
    Merged-In: I88cf7683e036ff43bbe3ff5418519c7e35e08b79
    (cherry picked from commit d0ab4e3701dcec866cb05f4d2a122fbbcabf61cc)

commit b8371d532d6cd1314954c2d3e9b8a79d3d70e7c7
Author: Hridya Valsaraju <hridya@google.com>
Date:   Thu May 31 12:39:58 2018 -0700

    Populate recovery DTBO offset correctly

    Also, remove recovery_dtbo_offset argument
    for mkbootimg as this is calculated based
    on sizes of kernel, ramdisk and second
    binaries.

    Also, modify unpack_bootimg to use the recovery_dtbo_offset
    field to extract recovery_dtbo.

    Test: make recoveryimage showcommands -j32
    Bug: 80207223
    unpack_bootimg --boot_img $OUT/recovery.img
    diff recovery_dtbo dtbo.img

    Change-Id: I588ccc8b739c169b6f78c17ffe554c5562397d98
    Merged-In: I588ccc8b739c169b6f78c17ffe554c5562397d98
    (cherry picked from commit 26e01bbdc5cb2dcad64d73ec836447b8ded4fc6c)

commit a612ed76f3d150c660ad3fd1d96245b3898aa4b5
Merge: 892e4f3f6 820ef1505
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Thu Jun 7 03:07:24 2018 +0000

    Snap for 4826407 from 820ef1505828e89b704f319311ed0cf4ec73752c to pi-dr1-release

    Change-Id: Iae002df17be08c1cc01373c1dbb0ac51c1e094c6

commit 892e4f3f6214dcf20304ff419f017ba83c7983f5
Merge: 9bd3ee1c4 ea03e1f84
Author: android-build-team Robot <android-build-team-robot@google.com>
Date:   Wed Jun 6 03:02:38 2018 +0000

    Snap for 4823496 from ea03e1f8412903838b449871190e1279dc79e076 to pi-dr1-release

    Change-Id: I8b3dc0bcb317027be08338bed1346315fba7bf87

Change-Id: Ic0a7952d369487aceb1f851d1c789d6ea3120542
---
 libmetricslogger/include/metricslogger/metrics_logger.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libmetricslogger/include/metricslogger/metrics_logger.h b/libmetricslogger/include/metricslogger/metrics_logger.h
index c305db2da..88575be9d 100644
--- a/libmetricslogger/include/metricslogger/metrics_logger.h
+++ b/libmetricslogger/include/metricslogger/metrics_logger.h
@@ -114,7 +114,7 @@ enum {
     FIELD_BATTERY_RESISTANCE_UOHMS = 1448,
     FIELD_BATTERY_CURRENT_UA = 1449,
     FIELD_HARDWARE_LOCATION = 1450,
-    ACTION_BATTERY_CAUSED_SHUTDOWN = 1441,
+    ACTION_BATTERY_CAUSED_SHUTDOWN = 1451,
 };
 
 enum {
-- 
2.17.1

